packages/plugins/other/visitor-plugin-common/src/base-types-visitor.ts

```diff
@@ -577,6 +596,24 @@ export class BaseTypesVisitor<
 
     const schemaType = this._schema.getType(node.name as any);
 
+    if (this.config.useImplementingInterfaces) {
+      const allTypesMap = this._schema.getTypeMap();
+      const implementingTypes: string[] = [];
+
+      for (const graphqlType of Object.values(allTypesMap)) {
+        if (graphqlType instanceof GraphQLObjectType) {
+          const allInterfaces = graphqlType.getInterfaces();
```

Comment:
Does this recursively handles interfaces that implement another interface?

Reply:
Good point. I'll add a unit test and check

Reply:
I added a test which should cover this case and it finishes without error. But please have another look, maybe I've missed something